scriptmodules/supplementary/audiosettings.sh

```diff
@@ -30,15 +30,18 @@ function gui_audiosettings() {
     # The list of ALSA cards/devices depends on the 'snd-bcm2385' module parameter 'enable_compat_alsa'
     # * enable_compat_alsa: true  - single soundcard, output is routed based on the `numid` control
     # * enable_compat_alsa: false - one soundcard per output type (HDMI/Headphones)
-    if aplay -l | grep -q "bcm2835 ALSA"; then
+    # If PulseAudio is enabled, then try to configure it and leave ALSA alone
+    if sudo -u "$user" XDG_RUNTIME_DIR=/run/user/$SUDO_UID systemctl -q --user is-enabled pulseaudio.socket; then
```

Comment:
I believe this will output an error line if pulseaudio isn't installed - probably don't want to show that

Reply:
Yeah it dumps a "Failed to get unit state for pulseaudio.socket" if not installed. Can be redirected.

Reply:
Yes, it errors out. Should be handled by the helper function.